### DISCUSSION TITLE
Amazon MemoryDB for Redis

### DISCUSSION THREAD

**Discussion by Fantros** [aws/aws-sdk-js-v3#3899] [Q&A]
How can we access it with sdk like redis? I've looked at the documentation and there's nothing about it.

**Comment by ajredniwja** [comment]
@Fantros thanks for opening this issue, I am not so familiar with redis but are you looking for a MemoryDb client? 
V3 client for MemoryDb: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-memorydb/index.html

Using console:
https://docs.aws.amazon.com/memorydb/latest/devguide/getting-started.html?pg=mdb&sec=hero-btn&refid=83add82a-8e52-4837-bc73-c323da62d78c
